addons/knobs/src/components/types/Files.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function fileReaderPromise(file) {
+  return new Promise(resolve => {
+    const fileReader = new FileReader();
+    fileReader.onload = e => resolve(e.currentTarget.result);
+    fileReader.readAsDataURL(file);
+  });
+}
+
+const FilesType = ({ knob, onChange }) => (
+  <input
+    id={knob.name}
+    type="file"
```

Comment:
It would be handy to be able to pass `accept` atribute here
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#Limiting_accepted_file_types